app/src/main/java/ch/eureka/eurekapp/ui/tasks/components/TaskSectionHeader.kt

```diff
@@ -0,0 +1,37 @@
+package ch.eureka.eurekapp.ui.tasks.components
+
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.font.FontWeight
+import ch.eureka.eurekapp.ui.designsystem.tokens.Spacing
+
+/** Section header for task lists Used for "Terminées récemment" and other sections */
+@Composable
+fun TaskSectionHeader(title: String, taskCount: Int? = null, modifier: Modifier = Modifier) {
+  Column(modifier = modifier.fillMaxWidth().padding(vertical = Spacing.sm)) {
+    Text(
+        text = title,
+        style = MaterialTheme.typography.titleMedium,
+        fontWeight = FontWeight.SemiBold,
+        color = MaterialTheme.colorScheme.onSurface)
+
+    if (taskCount != null) {
+      val taskText =
+          when (taskCount) {
+            0 -> "tâche"
+            1 -> "tâche"
+            else -> "tâches"
+          }
```

Comment:
why is this still in French also 0 takes plural also could be written as

```suggestion
      val taskText = "$taskCount task${if(taskCount != 1) "s" else ""}"
```